client/src/components/EventsModal.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <v-dialog fullscreen>
+    <v-card color="#ECECEC">
```

Comment:
lets config this colour into our global custom colours since its used on multiple pages